frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -84,7 +84,10 @@ describe('Kubernetes resource CRUD operations', () => {
         // sidebar needs to be fully loaded, else it sometimes overlays the Create button
         cy.byTestID('resource-sidebar').should('exist');
         yamlEditor.isLoaded();
-        cy.testA11y(`YAML Editor for ${kind}: ${name}`);
+        // YAML Editor same for all resources, so only a11y test once
+        if (kind === 'Pod') {
+          cy.testA11y(`YAML Editor for ${kind}: ${name}`);
+        }
```

Comment:
Is there a reason we shouldn't test for every resource? I could see us adding special handling for certain resource. We're already talking about adding a YAML/Form switch for some resources.

Reply:
I thought the YAML Editor html elements would be the same for every resource, only the inner YAML text changes, so no need to rerun a11y checks for every resource, just one would do, right?

Reply:
> I thought the YAML Editor html elements would be the same for every resource, only the inner YAML text changes, so no need to rerun a11y checks for every resource, just one would do, right?

I wouldn't assume that will always be true, and there's no virtually no overhead to checking a11y for each resource. And it makes the code simpler as well. I don't see a reason to restrict it to just pods.